packages/form-builder/src/feedback/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from "react";
+import {__} from '@wordpress/i18n';
+import Container from './Container'
+import FeedbackButton from "./FeedbackButton";
+
+import {
+    Container as PopupContainer,
+    Header as PopupHeader,
+    Content as PopupContent,
+} from './popup'
+
+const feedbackUrl = 'https://feedback.givewp.com/next-gen';
```

Comment:
Can we move this url to our `storageData` from the server.  It will just make it easier to update that Url if need be.

Reply:
This is a URL that _should not_ change - and if it did it would require a new release, which means a build. I'd rather maintain simplicity here as this component is temporary.